### ISSUE TITLE
Treat .spsql and .udfsql Differently from .sql and .ddl

### ISSUE THREAD

**Issue by scotdb** [IBM/db2forzosdeveloperextension-about#15]
It is great that you recognize the .spsql and .udfsql file extensions from Data Studio.   Very important for those of us with a large codebase in source control that we want to bring forward from Data Studio to VSCode.

There is however a significant difference between the .spsql and .udfsql files and .sql files in Data Studio.   These contain the code for EXACTLY ONE stored procedure or UDF.   And, most importantly, they don't have a terminator - since the single file is regarded as a unit in its own right.  That might seem to be a problem for automated deployment by other tools.   But it isn't if you use CLPPLUS, which is what we do (a single STA command per SP or UDF file).   

This "one file per SP / UDF" mantra goes deeper than that.  For example, if you change the SP name then the file name will be automatically changed to keep pace with this.   

Also there are a variety of smarts associated with the file - including debugging and deployment options and even the last values of the parameters used to execute or debug the code.      

Bottom line - I'd really like .spsql and .udfsql to continue to not need a terminator in the file, so that we can take all our code from Data Studio unchanged.   Also this will enable you for the future as you flesh out the functionality.

**Comment by realkendrickren** [comment]
@scotdb thanks a lot for bringing this up, this is also one of items we plan to support in future. Stay tuned.

**Comment by realkendrickren** [comment]
@scotdb  
Hi Philip we just published v1.1.0 a few minutes ago. This enhancement request is addressed now. Please upgrade the extension in VSCode and give a try. I now close this. Let me know if there is any question.

Thanks a lot again for using the extension, reporting issues/enhancements and posting your review comment on VSCode marketplace.
